src/webauth/utils.ts

```diff
@@ -133,6 +137,97 @@ export function buildChallengeTx(
     .toString();
 }
 
+/**
+ * Checks if a transaction has been signed by one or more of the given signers,
+ * returning a list of non-repeated signers that were found to have signed the
+ * given transaction.
+ *
+ * @function
+ * @memberof WebAuth
+ * @param {Transaction} transaction the signed transaction.
+ * @param {string[]} signers The signers public keys.
+ * @returns {string[]} a list of signers that were found to have signed the
+ * transaction.
+ *
+ * @example
+ * let keypair1 = Keypair.random();
+ * let keypair2 = Keypair.random();
+ * const account = new StellarSdk.Account(keypair1.publicKey(), "-1");
+ *
+ * const transaction = new TransactionBuilder(account, { fee: 100 })
+ *    .setTimeout(30)
+ *    .build();
+ *
+ * transaction.sign(keypair1, keypair2)
+ * WebAuth.gatherTxSigners(transaction, [keypair1.publicKey(), keypair2.publicKey()])
+ */
+export function gatherTxSigners(
```

Comment:
Did anything change in these? Since they're so security-sensitive I'm praying there are no bugs introduced by some sort of eslint-compatibility changes

Reply:
Mainly just reordering function calls so that the definitions come before they are called. Some for loops were changed, I'll make a separate PR for these.